src/mpack/conv.c

```diff
@@ -328,7 +327,6 @@ static mpack_value_t mpack_pack_ieee754(double v, unsigned mantbits,
   double mant;
 
   if (v == 0) {
-    rv.lo = 0;
     rv.hi = 0;
```

Comment:
won't it also complain about rv.hi here?

Typically we leave the declaration uninitialized. That forces the fields to be set explicitly in each codepath.

Reply:
That's... a good question. I have no idea why PVS didn't warn about `rv.hi`.

Didn't quite follow how I should solve this. Could you elaborate a bit more?

Reply:
My suggestion was to change `mpack_value_t rv = {0, 0};` to just `mpack_value_t rv;` .

Reply:
Gotcha.